### DISCUSSION TITLE
Static cache invalidator works differently with database queue

### DISCUSSION THREAD

**Discussion by DavidKrenekCZ** [statamic/cms#4737] [Troubleshooting]
Hi, I found something strange. I'm using full static caching for whole page. When I have set `sync` queue connection, everything works fine and pages are invalidating by rules that I set in `static_caching.php` config file. But if I change queue connection to `database` with worker, in some cases invalidating works fine, but sometimes after entry save is static cache invalidate whole content even though it's not needed or set up in any way in `static_caching.php`. In general - the same cache invalidation setting works differently with `sync` and `database` queue connections. Any idea where the problem might be please?

**Comment by edalzell** [comment]
Likely this: https://github.com/statamic/cms/issues/3291
